src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/DocumentValidationController.java

```diff
@@ -48,8 +48,7 @@
             @NotNull @RequestBody @ApiParam("CaseData") CallbackRequest callbackRequest,
             @PathVariable("field") String field) {
 
-        long caseId = callbackRequest.getCaseDetails().getId();
-        log.info("Received request for checkUploadedFileType for Case ID: {}", caseId);
+        log.info("Received request for checkUploadedFileType");
```

Comment:
is it better than having the case ID printed as null in logs? is it always null in this case?

Reply:
This issue is fixing the occurrence where it is a case submission - before the case number has been created. This should be in the only place in the system where this occurs (as the Case hasnt been submitted and does not have an ID)